web/js/helpers/requests.js

```diff
@@ -103,20 +103,6 @@
     return sendXHR('DELETE', `${server}/archive/${id}`);
   }
 
-  function saveConnection(connection, sessionId) {
-    return sendXHR('POST', `${server}/saveConnection/`, JSON.stringify({
-      connection,
-      sessionId
-    }), 'application/json');
-  }
-
-  function deleteConnection(connection, sessionId) {
-    return sendXHR('POST', `${server}/deleteConnection/`, JSON.stringify({
-      connection,
-      sessionId
-    }), 'application/json');
-  }
```

Comment:
There is still a reference to `Request.deleteConnection` in roomStatus.js.

Reply:
sorry, thxs